src/components/Form/Form.tsx

```diff
@@ -22,19 +22,54 @@ const Form = ({ updateUserSentiment }: FormProps): JSX.Element => {
 
   return (
     <section className="questionnaire">
-      <h2 className='cy-sentiment-question'>how are you feeling today?</h2>
-      <div className="sentiment-selection">
-        <div className="sentiments-container">
-          <button className="happy-btn cy-happy-btn" name='1' aria-label="happy" title="happy" onClick={() => submitUserSentiment(1)}></button>
-          <button className="neutral-btn cy-neutral-btn" name='0' aria-label="neutral" title="neutral" onClick={() => submitUserSentiment(0)}></button>
-          <button className="sad-btn cy-sad-btn" name='-1' aria-label="sad" title="sad" onClick={() => submitUserSentiment(-1)}></button>
-        </div>
+      <h2 className='cy-sentiment-question'>What kind of mood are you in?</h2>
+      <div className="sentiments-container">
+        <button
+          className="sentiment-btn cy-completely-positive-btn"
+          aria-label="strongly positive"
+          title="strongly positive"
+          onClick={() => submitUserSentiment(1)}
+        >
+          😁
+        </button>
+        <button
+          className="sentiment-btn cy-positive-btn"
+          aria-label="positive"
+          title="positive"
+          onClick={() => submitUserSentiment(0.5)}
+        >
+          🙂
+        </button>
+        <button
+          className="sentiment-btn cy-neutral-btn"
+          aria-label="neutral"
+          title="neutral"
+          onClick={() => submitUserSentiment(0)}
+        >
+          😶
+        </button>
+        <button
+          className="sentiment-btn cy-negative-btn"
+          aria-label="negative"
+          title="negative"
+          onClick={() => submitUserSentiment(-0.5)}
+        >
+          🙁
+        </button>
+        <button
+          className="sentiment-btn cy-completely-negative-btn"
+          aria-label="strongly negative"
+          title="strongly negative"
+          onClick={() => submitUserSentiment(-1)}
+        >
+          ☹️
+        </button>
```

Comment:
Is it just me, or would it seem more natural for the sentiments to be reversed, that is, from completely negative on the left to completely positive on the right?  Our scale is from 0-10 (negative to positive). Since we usually see a 0-10 scale set out with 0 on the left and 10 on the right, our emoji scale (which is based on the 0-10 scale) should follow the same pattern. This is especially so if we display our 0-10 indicator on each article.
```suggestion
        <button
          className="sentiment-btn cy-completely-negative-btn"
          aria-label="strongly negative"
          title="strongly negative"
          onClick={() => submitUserSentiment(-1)}
        >
          ☹️
        </button>
        <button
          className="sentiment-btn cy-negative-btn"
          aria-label="negative"
          title="negative"
          onClick={() => submitUserSentiment(-0.5)}
        >
          🙁
        </button>
        <button
          className="sentiment-btn cy-neutral-btn"
          aria-label="neutral"
          title="neutral"
          onClick={() => submitUserSentiment(0)}
        >
          😶
        </button>
         <button
          className="sentiment-btn cy-positive-btn"
          aria-label="positive"
          title="positive"
          onClick={() => submitUserSentiment(0.5)}
        >
          🙂
        </button>
        <button
          className="sentiment-btn cy-completely-positive-btn"
          aria-label="strongly positive"
          title="strongly positive"
          onClick={() => submitUserSentiment(1)}
        >
          😁
        </button>
```